packages/material-ui/src/Popover/Popover.js

```diff
@@ -331,7 +333,6 @@ class Popover extends React.Component {
           onExit={onExit}
           onExited={onExited}
           onExiting={onExiting}
-          role={role}
```

Comment:
Not sure why we would want the `role` on the TransitionComponent. There was no test for it nor internal usage. Change that added this is missing.

Reply:
This logic hasn't changed since the first commit. I'm happy to remove it 👍